src/event/pos_event.yaml

```diff
@@ -1228,6 +1228,14 @@ Root:
     Cause: State of the input array is not online.
     Solution:
 
+  -
+    Id: 1878
+    Name: LIST_VOL_ARRAY_NAME_DOES_NOT_EXIST
```

Comment:
LIST_VOLUME_ ... can be better for consistency with others?

Reply:
As discussed, This is consistent with other events name 
Thanks